corehq/motech/repeaters/migrations/0001_adjust_auth_field_format_squashed_0015_drop_connection_settings_fk.py

```diff
@@ -336,4 +367,5 @@ class Migration(migrations.Migration):
             },
             bases=('repeaters.createcaserepeater',),
         ),
+        RepeaterDbSQL(REPEATER_SEQ, UNDO_REPEATER_SEQ),
```

Comment:
Seems odd to add this to an existing migration but I assume it's a short cut that works because you're setting up a new DB and this operation should only run on the new DB right?

Reply:
Correct.